src/pkg/cli/deploymentsList.go

```diff
@@ -37,12 +39,18 @@ func DeploymentsList(ctx context.Context, loader client.Loader, client client.Gr
 	// map to Deployment struct
 	deployments := make([]PrintDeployment, numDeployments)
 	for i, d := range response.Deployments {
+		actionText := strings.TrimPrefix(d.Action.String(), "DEPLOYMENT_ACTION_")
+		if d.Action == defangv1.DeploymentAction_DEPLOYMENT_ACTION_UNSPECIFIED {
```

Comment:
Let's amend the `ListDeployments` query to exclude deployment actions which are not `"UP"`